programs/create-genesis/ee_genesis/event_engine_genesis.cpp

```diff
@@ -65,6 +65,78 @@ static abi_def create_transfers_abi() {
             {"to", "name"},
             {"quantity", "asset"},
             {"memo", "string"},
+            {"to_vesting", "bool"},
+            {"time", "time_point_sec"},
+        }
+    });
+
+    return abi;
+}
+
+static abi_def create_withdraws_abi() {
+    abi_def abi;
+    abi.version = ABI_VERSION;
+
+    abi.structs.emplace_back( struct_def {
+        "withdraw", "", {
+            {"from", "name"},
+            {"to", "name"},
+            {"quantity", "asset"},
+            {"time", "time_point_sec"},
+        }
+    });
+
+    return abi;
+}
+
+static abi_def create_delegations_abi() {
+    abi_def abi;
+    abi.version = ABI_VERSION;
+
+    abi.structs.emplace_back( struct_def {
+        "delegate", "", {
+            {"from", "name"},
+            {"to", "name"},
+            {"quantity", "asset"},
+            {"interest_rate", "uint16"},
+            {"payout_strategy", "uint8"},
```

Comment:
we have no payout strategy in contracts, and it's limited in Golos to one value now. do we need it here?